packages/core/src/components/ChoiceList/Choice.jsx

```diff
@@ -11,41 +11,88 @@ import uniqueId from 'lodash.uniqueid';
  * to the `input` element, so you can use this to set additional attributes if
  * necessary.
  */
-export const Choice = function(props) {
```

Comment:
As I mention in the PR description, an easier way to view the diff for this file is to view the diff with whitespace ignored. You can view that diff here: https://github.com/CMSgov/design-system/pull/173/files?w=1